examples/cisco/demo_vxlan.pp

```diff
@@ -49,4 +49,10 @@
     multicast_group     => '224.1.1.1',
     suppress_arp        => 'default',
   }
+
+  cisco_interface { 'vlan97':
+    ensure => present,
+    fabric_forwarding_anycast_gateway => 'true',
+    require                           => Cisco_overlay_global['default'],
+  }
```

Comment:
`demo_vxlan.pp` is only included on supported platforms, but I need to update this to only include n9k.

https://github.com/cisco/cisco-network-puppet-module/blob/release_1.2.0/examples/demo_all_cisco.pp#L46